src/odemis/gui/util/img.py

```diff
@@ -1553,30 +1553,35 @@ def _get_stream_legend_text(s):
 
     # FluoStreams are merged using the "Screen" method that handles colour
     # merging without decreasing the intensity.
-    data_raw = s.raw[0]
+    if isinstance(s.raw, list):
+        md = s.raw[0].metadata
+    elif isinstance(s.raw, model.AcquisitionData):
```

Comment:
There is no reason (anymore) that .raw can ever be a AcquisitionData. => `elif isinstance(s.raw, (DataArray, DataArrayShadow)):`